CONTRIBUTING.md

```diff
@@ -41,12 +42,12 @@ Getting your contributions integrated is relatively straightforward, here
 is the checklist:
 
 * All tests pass
-* Consensus is reached. This basically mean that the reviewers add `LGTM` comments. `LGTM` is a
+* Consensus is reached. This basically means that the reviewers add `LGTM` comments. `LGTM` is a
```

Comment:
@zachmullen Do you mean we should go with `Consensus is reached. This usually means that at least one reviewer added a `LGTM` comment and reasonable amount of time went without anyone objecting.` or keep the older version ?